plugins/woocommerce/package.json

```diff
@@ -225,6 +226,26 @@
 						"push"
 					]
 				},
+				{
```

Comment:
It's my first time using this new format for tests.  Double check my logic? On release checks and as part of the daily run.

Reply:
Joining the leaning club: how the `config .ci` section works in general?